app/src/main/java/com/tennistourcol/ui/tournament/EditTorunamentFragment.java

```diff
@@ -58,7 +71,37 @@ public void onCreate(Bundle savedInstanceState) {
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.fragment_edit_torunament, container, false);
         // Inflate the layout for this fragment
-        return inflater.inflate(R.layout.fragment_edit_torunament, container, false);
+        Retrofit retrofit = new Retrofit.Builder()
+                .baseUrl("https://ieti-back.herokuapp.com/") //localhost for emulator
+                .addConverterFactory(GsonConverterFactory.create())
+                .build();
+        tournamentService = retrofit.create(TournamentService.class);
+        ((FloatingActionButton) view.findViewById(R.id.deleteTournamentButton)).setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+//                deleteTournament(Tournament.builder().id("1").build());
+                System.out.println("ELIMINAR-----------------");
```

Comment:
Do not use the standard output for debugging messages. Consider configuring a proper logger.